odoo/README-short.txt

```diff
@@ -0,0 +1 @@
+Odoo (formerly known as OpenERP) is a suite of open-source business apps.
```

Comment:
Ohhh, Odoo literally is the continuation of the OpenERP work?  I thought it was more of a wrapper/add-on, but if it's a fork/rename that makes much more sense. :+1:

> Odoo was formerly known as OpenERP until May 2014. It was rebranded because version 8 of the software included apps including website builder, e-commerce, point of sale and business intelligence. The software conforms to standard expectations of ERP systems, while providing additional modules beyond the coverage of traditional ERP systems.